CockpitPlugin/build.gradle

```diff
@@ -39,6 +40,11 @@ sourceSets {
     }
 
     test.java.srcDirs += 'src/test/kotlin'
+    main.java.srcDirs += project(':CockpitCore').file('src/main/java')
+}
+
+jar {
+    from zipTree('../CockpitCore/build/libs/CockpitCore-1.1.1-SNAPSHOT.jar')
```

Comment:
what about extracting core version to some variable and use `$core_version` here?